docs/csharp/language-reference/keywords/out-parameter-modifier.md

```diff
@@ -41,6 +41,8 @@ You can't use the `in`, `ref`, and `out` keywords for the following kinds of met
   
 - Iterator methods, which include a [yield return](./yield.md) or `yield break` statement.  
 
+- Extension methods, where the first argument has the `this` modifier.
```

Comment:
I think you mean the first argument only? It sounds like you mean you can't use these at all with extension methods.